atomic_reactor/core.py

```diff
@@ -579,6 +623,7 @@ def remove_container(self, container_id, force=False):
         logger.debug("container_id = '%s'", container_id)
         self.d.remove_container(container_id, force=force)  # returns None
 
+    @retry
     def logs(self, container_id, stderr=True, stream=True):
```

Comment:
This one's interesting, because it's a generator function. Calling the function will always succeed and return a generator; it's when iterating through it that API calls may fail. I don't think wrapping this generator function by `retry` will have any effect.